src/test/java/com/develhope/spring/vehicles/entities/VehicleTests.java

```diff
@@ -11,37 +11,4 @@
 
 public class VehicleTests {
```

Comment:
Fixa i conflitti, e rimetti in play manualmente i test uno per uno, non tutta la classe altrimenti non funzionano, e assicurati che passino di nuovo tutti dopo le modifiche, non voglio brutte più brutte sorpresexD

Reply:
comunque mi so sbagliato va messa in play tutta la classe, ma spero funzionassero comunque